helm_deploy/create-and-vary-a-licence-api/values.yaml

```diff
@@ -31,6 +32,11 @@ generic-service:
   namespace_secrets:
     create-and-vary-a-licence-api:
       APPINSIGHTS_INSTRUMENTATIONKEY: "APPINSIGHTS_INSTRUMENTATIONKEY"
+    rds-instance-output:
+      DB_SERVER: "rds_instance_address"
+      DB_NAME: "database_name"
+      DB_USER: "database_user"
+      DB_PASS: "database_password"
 
   allowlist:
```

Comment:
Adding the database secrets to et environment variables by reading values from the kubernetes secret called `rds_instance_output` in the dev namespace.